### DISCUSSION TITLE
Roadmap for 1.0.0 Release

### DISCUSSION THREAD

**Discussion by Tommyten** [arkivanov/Decompose#109] [General]
Hey @arkivanov,

I was wondering whether there was a Roadmap for a first major Release?
If so, could you give us an idea, of what we can expect and how we can help achieving this goal?

I think having a first major release could help some people in deciding to adopt Decompose for their projects, as it signals that there is a certain level of accountability.

**Comment by arkivanov** [comment]
Thanks for raising this! Basically, the main API (`decompose` module) already looks stable. However, there are few things left to be done before the release. I have filed #112.
